clients/src/main/java/org/apache/kafka/clients/admin/CreateAclsResult.java

```diff
@@ -41,7 +41,7 @@
      * Return a map from ACL bindings to futures which can be used to check the status of the creation of each ACL
      * binding.
      */
-    public Map<AclBinding, KafkaFuture<Void>> results() {
+    public Map<AclBinding, KafkaFuture<Void>> value() {
```

Comment:
For example, if you had a method which returned an array of message objects, you would probably call that `messages` rather than `message`, even though technically the array you returned is a single object.  The important thing is that that array contains multiple values.